includes/currency-functions.php

```diff
@@ -588,11 +609,11 @@ function give_currency_filter( $price = '', $currency_code = '', $decode_currenc
 	 *           filter name will be give_usd_currency_filter_after
 	 */
 	$formatted = apply_filters(
-		'give_' . strtolower( $currency_code ) . "_currency_filter_{$position}",
+		'give_' . strtolower( $args['currency_code'] ) . "_currency_filter_{$position}",
 		$formatted,
-		$currency_code,
+		$args['currency_code'],
 		$price,
-		$decode_currency
+		$args
```

Comment:
@emgk I pass all argument to this filter, so it will be useful for you.

Reply:
Yes, that will be useful, Thanks @ravinderk